src/home_robot/home_robot/motion/stretch.py

```diff
@@ -157,8 +159,11 @@ class HelloStretchKinematics(Robot):
     look_ahead = np.array([0.0, 0.0])
 
     # For inverse kinematics mode
-    ee_link_name = "link_straight_gripper"
-    manip_mode_controlled_joints = [
+    default_ee_link_name = "link_straight_gripper"
+
+    # TODO: is there a particular reason the arm ls are inverted from PIN_CONTROLLED_JOINTS?
```

Comment:
Is there a particular reason the arm ls are inverted from PIN_CONTROLLED_JOINTS? If not, I think we can just use PIN_CONTROLLED_JOINTS instead of re-defining?